target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -60,9 +60,7 @@ ipq40xx_setup_interfaces()
 		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4" "wan"
 		;;
 	zte,mf289f)
-		ucidef_set_interfaces_lan_wan "eth0" "eth1"
-		ucidef_add_switch "switch0" \
-			"0u@eth0" "0u@eth1" "2:lan:1" "5:lan:2"
+		ucidef_set_interfaces_lan_wan "lan" "wan"
```

Comment:
There is an existing section with the same config, just add this device to it.

Reply:
done